plan/plan.go

```diff
@@ -76,6 +79,9 @@ func (p *Plan) Calculate() *Plan {
 		}
 	}
 
+	// Apply policy to list of changes.
+	changes = *p.Policy.Apply(&changes)
```

Comment:
This call looks complicated. You pass a ptr to changes and it returns a ptr to changed "changes" I guess which you then dereference.
For me it looks like it would be simpler to have changes.Apply(p.Policy) manipulating changes.